DESCRIPTION

```diff
@@ -23,14 +27,17 @@ Authors@R:
 License: MIT + file LICENSE
```

Comment:
@jaeltan you can add yourself to the end of this list once you're done with this PR